githooks.sh

```diff
@@ -891,26 +918,26 @@ update_release_clone() {
     GITHOOKS_CLONE_URL=$(git config --global githooks.autoupdate.updateCloneUrl)
     GITHOOKS_CLONE_BRANCH=$(git config --global githooks.autoupdate.updateCloneBranch)
 
-    if is_git_repo "$CLONE_DIR"; then
-        URL=$(git -C "$CLONE_DIR" config remote.origin.url)
-        BRANCH=$(git -C "$CLONE_DIR" symbolic-ref -q --short HEAD)
+    if is_git_repo "$GITHOOKS_CLONE_DIR"; then
+        URL=$(execute_git "$GITHOOKS_CLONE_DIR" config remote.origin.url)
+        BRANCH=$(execute_git "$GITHOOKS_CLONE_DIR" symbolic-ref -q --short HEAD)
 
         if [ "$URL" != "$GITHOOKS_CLONE_URL" ] ||
             [ "$BRANCH" != "$GITHOOKS_CLONE_BRANCH" ]; then
-            echo "! Cannot pull updates because \`origin\` of update clone \`$CLONE_DIR\` points" >&2
+            echo "! Cannot pull updates because \`origin\` of update clone \`$GITHOOKS_CLONE_DIR\` points" >&2
             echo "  to url:" >&2
             echo "   \`$URL\` on branch \`$BRANCH\`" >&2
             echo "  which is not configured." >&2
             echo "  See \`git hooks config [set|print] update-clone-url\` and" >&2
             echo "      \`git hooks config [set|print] update-clone-branch\`" >&2
-            echo "  Either fix this or delete the clone \`$CLONE_DIR\` to trigger" >&2
+            echo "  Either fix this or delete the clone \`$GITHOOKS_CLONE_DIR\` to trigger" >&2
```

Comment:
No scripts outside the clone dir. This githooks.sh is running and will get removed, which is possible because the shell interpreted has already sourced this whole script under linux this does not pose a problem so far, we rely on the behavior that we can rm running scripts (headache)... on windows I hope it does not break. it should not with git-bash...

Reply:
Huh? But this echo instructs the user to remove the directory, right? Suggesting that that should subsequently trigger a new clone? But when the user deletes that directory, no scripts are running, and on the next git push or whatever, the hooks will just fail because  `githooks.sh` was deleted?